CHANGELOG.md

```diff
@@ -3,6 +3,50 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 5.0.0-beta.7
+
+_Nov 4, 2021_
+
+- 👁 Move virtualization logic to hook (#3079) @m4theushw
```

Comment:
It is 100% internal, not sure we should highlight it

Reply:
Definitely no.

```suggestion
- 💅 Reduce styles specificity to make simpler to override (#3012) @DanailH
```